src/docs/guides/23-how-to-use-custom-pricing/index.md

```diff
@@ -0,0 +1,793 @@
+---
+title: How to customize pricing
+slug: How to customize pricing
+updated: 2019-03-13
+category: guides
+ingress:
+  Flex allows lots of flexibility for your providers in terms of how
+  they can set their pricing. This guide walks you through customizing
+  listing pricing and processing the pricing calculations correctly
+  during the transactions.
+published: true
+---
+
+## What kind of pricing can you achieve with Flex
+
+Here are some examples of common pricing schemes you might want to add
+on top of the regular listing price.
+
+- **Add-ons.** Upsell additions on top of the regular price. Examples:
+  cleaning fee, insurance, delivery.
+- **Discount** based on booking length. An example: daily price is
+  $20, weekly price
+  $70, and monthly price \$200. Another example: 20%
+  discount on daily rate for bookings of 5 or more days, 30% discount on
+  bookings of 10 or more days, and so on.
+- **Taxes.** Add any type of tax rates to the listing price and display
+  them as separate line items in the receipt.
+- **Offer multiple pricing packages in the same listing.** An example: a
+  hairdresser offers both haircuts and hair dying.
+- **Seasonal pricing.** Examples: weekends cost more than weekdays,
+  summers cost more than winters, and so on.
+- **Quantity discount.** An example: booking a room for two people costs
+  $100, and
+  after that each additional person costs $20 extra.
+
+All of these can be achieved by storing pricing information in the
+[extended data](/references/extended-data/) of the listings, and then
+adjusting your [transaction process](/background/transaction-process/)
+by adding a new action called set-line-items-and-total. As the name
+suggests, this action allows you to set any number of line items you
+need (based on the price information stored in extended data and the
+choices the customer makes when initiating the booking) and calculate
+the total price of the transaction based on them.
+
+This guide walks you through the process of using the
+set-line-items-and-total action by using the example of a common pricing
+adjustment: adding the ability to charge an optional cleaning fee on top
+of the regular nightly price of the accommodation.
+
+## 1. Process change
+
+To get started, the transaction process needs to be updated to support
+custom pricing. The requierd change is to add and action called
+`set-line-items-and-total` to all the transitions that create a booking.
+For this, you need to
+[contact Sharetribe support](mailto:flex-support@sharetribe.com) and ask
+them to make the process adjustment for you. In this guide, we're
+assuming that the `set-line-items-and-total` action has been added to
+the `transition/request` and `transition/request-after-enquiry`
+transitions in the default transaction process.
+
+Note: `set-line-items-and-total` cannot be combined with other pricing
```

Comment:
Did you try blockquote syntax: `> Note:`
I have used that for side notes. (Although, I'm not sure if that should be used here.)

Ref:
https://www.sharetribe.com/docs/guides/how-to-change-transaction-process-in-ftw/#1-change-the-transaction-process-alias